test/atomic_integer_test.rb

```diff
@@ -0,0 +1,57 @@
+require 'test_helper'
+
+class TestAtomicInteger < MiniTest::Unit::TestCase
+  def test_operations
```

Comment:
When doing a lot of assertions in one test in a unit test, that's usually a smell you could have more well-named tests. Note that one of those may be exactly equal to this one, but has a good name on what you're trying to test and perhaps only one or two assertions.